scripts/serve/serve.js

```diff
@@ -1,25 +1,43 @@
-const serve = require('webpack-serve');
+/* eslint-disable no-console */
+const WebpackDevServer = require('webpack-dev-server');
+const Webpack = require('webpack');
 
-const webpackServe = (options) => {
+// Create a webpack dev server instance.
+const server = (options) => {
   const { port, host } = options;
   let config = options.config;
+  // if config is a function, execute it with prod mode if applicable.
   if (typeof config === 'function') {
     config = config(undefined, { p: options.production });
   }
 
-  const serveConfig = {
-    config,
-    ...(port) && { port },
-    ...(host) && { host },
-    dev: {
-      stats: {
-        colors: true,
-        children: false,
-      },
+  // pull the dev server options out of the webpack config. override host, port, and stats. SRY.
+  const devServerOptions = Object.assign({}, config.devServer, {
```

Comment:
Lets add a fallback in case `config.devServer` is undefined

Reply:
ooh good call

Reply:
Actually this is fine. Object.assign handles devServer not being present.